config/locales/en/lookup.yml

```diff
@@ -18,7 +18,18 @@ en:
       body_content: |
         <p class="govuk-body">Enter the postcode of the place you want to find information about. For example where you live, work, or visit.</p>
         <p class="govuk-body">Each area has a <a class="govuk-link" href="/guidance/local-covid-alert-levels-what-you-need-to-know">Local COVID Alert Level</a>. There are 3 Local COVID Alert Levels. Sometimes these are called ‘tiers’ or known as a ‘local lockdown’.</p>
-        <p class="govuk-body">You can find out what you can or cannot do.</p>
+        <p class="govuk-body govuk-!-margin-bottom-7">You can find out what you can or cannot do.</p>
```

Comment:
Why has a margin class been added to the content here? What happens if a content designer accidentally removes it?

Reply:
It's a good question - markup could be removed entirely from this file. This was adding onto an existing class for speed. 

This could be re-factored entirely including removing `<p class="govuk-body`

Reply:
I think it's ok to keep the `<p class="govuk-body>` as all of the lines use it. However if the lines need to be re-ordered, this could cause a problem.

Reply:
Yeah, potentially it's the same risk, for the context of this it has been removed, worth revisiting.